demos/ASP.Net Core API Gateway/Divergent.Sales.API.Host/Controllers/OrdersController.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Divergent.Sales.Data.Context;
+using System.Collections.Generic;
+using System.Data.Entity;
+using System.Linq;
+using System.Web.Http;
+
+namespace Divergent.Sales.API.Host.Controllers
+{
+    [RoutePrefix("api/orders")]
+    public class OrdersController : ApiController
+    {
+        [HttpGet]
+        public dynamic Get(int id)
```

Comment:
Good point, probably back in time the return type was an anonymous class an di we never changed the action return type, that I should always serilized.
I'll fix it. Thanks.